common/achievement/index.ts

```diff
@@ -101,7 +100,14 @@ async function isAchievementConditionMet(achievement: UserAchievementTemplate) {
         return;
     }
 
-    const { conditionIsMet, resultObject } = await evaluateAchievement(condition, conditionDataAggregations as ConditionDataAggregations, metrics, recordValue);
+    const { conditionIsMet, resultObject } = await evaluateAchievement(
+        condition,
+        conditionDataAggregations as ConditionDataAggregations,
+        metrics,
+        recordValue,
+        achievement.userId,
+        achievement.context
```

Comment:
why do we not destructre the context and userId from the achievement - like the recordValue i.e.

Reply:
<img src="https://media2.giphy.com/media/A928XcIImHJAc/giphy.gif"/>